.vscode/settings.json

```diff
@@ -0,0 +1,9 @@
+{
+  "window.zoomLevel": 0,
```

Comment:
The lines which edit the editor appearance of vscode should be removed in order to not modify other developers editor.